ansible/playbook/README.adoc

```diff
@@ -147,6 +152,83 @@ For instance, to undo the previous host operation:
 ansible-playbook ansible/playbook/passstore_manage_host_groups.yml -e operation=remove -e group_name=k8s_115 -e vm_name=n01-k115
 ```
 
+=== update_letsencrypt_certificate
+
+[.lead]
+Updates the certificates for the openshift ingress console.
+
+.Playbook parameters
+[%header,cols="25%,75%"]
+|===
+| Variable | Description
+
+| `cluster_type`
+
+[.fuchsia]#string#
+
+a| Type of cluster where the certificate will be deployed into (target).
+
+* *kubernetes*
+* *openshift => Default*
+
+| `site_name`
+
+[.fuchsia]#string# / [.red]#required# 
+
+a| Name of the site that will be used to fetch the certificates from.
+
+The name of the site will be used to fetch the certificate from the 
+ `snowdrop-site` namespace under the format of `<site_name>-snowdrop-dev-tls`.
+
+| `source_host`
+
+[.fuchsia]#string#
+
+a| Name of the host that will be used to connect to the source cluster.
+
+*Default => `localhost`*
+
+| `source_kubeconfig`
+
+[.fuchsia]#string#
+
+a| `kubeconfig` file for the cluster running the license update feature.
+
+*Default => `${HOME}/.kube/config`*
+
+| `target_host`
+
+[.fuchsia]#string#
+
+a| Name of the host that will be used to connect to the source cluster.
+
+*Default => `localhost`*
+
+| `target_kubeconfig`
+
+[.fuchsia]#string#
+
+a| `kubeconfig` file for the cluster running the license update feature.
+
+*Default => `${HOME}/.kube/config`*
+
+|===
+
+Sample execution of the `update_letsencrypt_certificate.yml` playbook.
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/update_letsencrypt_certificate.yml \
+  -e source_kubeconfig=${HOME}/.kube/snowdrop-rhosp-snowdrop-k8s-config \ <1>
+  -e target_kubeconfig=${HOME}/.kube/rh-ocp-qshift-drp2b-config \ <2>
```

Comment:
Is it documented how users can fetch such config files ?

Reply:
Updated the documentation.